### ISSUE TITLE
Logo for application

### ISSUE THREAD

**Issue by Vishesht27** [DSCVITBHOPAL/Timer#1]
Add a logo for the entire application based on the theme that is Time

**Comment by KaranSinghBisht** [comment]
Hey I'd Like to Work On this, can you emphasize the theme and what does the app is about ?

**Comment by Vishesht27** [comment]
The app is simply using static data for now but in future work as a stopwatch . SO...

**Comment by KaranSinghBisht** [comment]
Oh okay. This is what I came up with till now. ![TimerLogo.png](https://user-images.githubusercontent.com/69008866/135712928-cb8139a3-328d-47a0-b0fa-6a19c0dd6deb.png)

**Comment by Dhruv-194** [comment]
> Oh okay. This is what I came up with till now. ![TimerLogo.png](https://user-images.githubusercontent.com/69008866/135712928-cb8139a3-328d-47a0-b0fa-6a19c0dd6deb.png)

Yes that's a good logo now send a PR by adding this logo into the "Images" folder of this repository.

**Comment by Shreyashmohadikar** [comment]
Hey!! I would like to contribute to this issue by designing a logo that will sit perfectly with the theme and working of the app(Stopwatch).
Can you assign me this issue?

**Comment by Dhruv-194** [comment]
> Hey!! I would like to contribute to this issue by designing a logo that will sit perfectly with the theme and working of the app(Stopwatch). Can you assign me this issue?

Hey @Shreyashmohadikar this issue is already been taken and we have also received the PR for it. You can start working on other issues which we have created. You can also create more issues for yourself to work on.
